modules/cnf-cpu-infra-container.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies:
+//
+// scalability_and_performance/cnf-performance-addon-operator-for-low-latency-nodes.adoc
+
+[id="cnf-cpu-infra-container_{context}"]
+= Restricting CPUs for infra and application containers
+
```

Comment:
Much of the text in the following paragraphs [existed in the PAO docs](https://docs.openshift.com/container-platform/4.7/scalability_and_performance/cnf-performance-addon-operator-for-low-latency-nodes.html#cnf-partitioning-the-cpus_cnf-master). I didn't want to change the text more than necessary to avoid changing the meaning or changing the flow of the assembly.